.github/workflows/UploadCoverageReport.yml

```diff
@@ -0,0 +1,33 @@
+---
+name: UploadCoverageReport
+
+'on':
```

Comment:
Is there any particular reason this is quoted?

Reply:
There is a `yamllint` rule [`truthy`](https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.truthy). yaml formatters change `on:` to `'on':`.